tests/test_meta/test_outlier_remover.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from pandas.tests.extension.numpy_.test_numpy_nested import np
+from sklearn.utils import estimator_checks
+
+from sklego.common import flatten
+from sklego.meta import OutlierRemover
+from sklego.mixture import GMMOutlierDetector
+
+
+@pytest.mark.parametrize("test_fn", flatten([
+    estimator_checks.check_transformers_unfitted,
+    estimator_checks.check_fit2d_predict1d,
+    estimator_checks.check_fit2d_1sample,
+    estimator_checks.check_fit2d_1feature,
+    estimator_checks.check_fit1d,
+    estimator_checks.check_get_params_invariance,
+    estimator_checks.check_set_params,
+    estimator_checks.check_dont_overwrite_parameters,
+    estimator_checks.check_transformers_unfitted
+]))
+def test_estimator_checks(test_fn):
+    outlier_remover = OutlierRemover(outlier_detector=GMMOutlierDetector(), refit=True)
```

Comment:
might be nice to also add `IsolationForest` here as well from normal `sklearn`.

Reply:
Sure!